### ISSUE TITLE
Possible bug with simple_sphere axis

### ISSUE THREAD

**Issue by dmishins** [BruceSherwood/glowscript#204]
Hi,
I believe there may be a bug with the simple_sphere object when it is set to a non-standard axis.  The following code produces a vertically compressed ellipsoid, when I believe the object should still be spherical:
```
GlowScript 2.7 VPython

ball = simple_sphere(radius = 4)

ball.axis = vec(sin(0),cos(0),0)

**Comment by BruceSherwood** [comment]
You're right. The simple_sphere should indeed follow the rules for sphere, namely that changing axis should not affect size (unlike most objects such as box or arrow). Thanks for the report. Of course it must be rather rare to change the axis of a simple_sphere. What is the situation to needing to change the axis?

**Comment by dmishins** [comment]
I posted more details about my goals in the google glowscript forums.  I tried to use a simple sphere hoping to get the animation to display smoothly, but it didn't seem to help so I returned to using a regular sphere.

**Comment by BruceSherwood** [comment]
Understood. Thanks. I'll fix the simple_sphere bug.

**Comment by BruceSherwood** [comment]
Fixed by commit #205
